tests/src/Common/CoreCLRTestLibrary/TestFramework.cs

```diff
@@ -24,25 +26,17 @@ public static void LogError(string id, string msg)
 
         public static void BeginTestCase(string title)
         {
-            int seed;
-#if WINCORESYS
-            seed = 20010415;
-#else
-            Random rand = new Random();
+            int seed = DEFAULT_SEED;
 
             if (Environment.GetEnvironmentVariable("CORECLR_SEED") != null)
             {
-                try
+                string CORECLR_SEED = Environment.GetEnvironmentVariable("CORECLR_SEED");
+
+                if (!int.TryParse(CORECLR_SEED, out seed))
                 {
-                    seed = int.Parse(Environment.GetEnvironmentVariable("CORECLR_SEED"));
+                    seed = DEFAULT_SEED;
```

Comment:
I think it would be worthwhile to allow some special value (like `RANDOM`) to indicate that a truly random seed is desired. This allows testing locally with a range of inputs, without needing to update the environment variable every time.

Does this sound reasonable?

Reply:
Seems reasonable to me. Are you suggesting a special value of `CORECLR_SEED`, or a separate environment variable. Maybe use the value of 0 for `CORECLR_SEED` to indicate a "truly" random seed, and never use an actual value of 0. Or something like that ...

Reply:
I was actually thinking just something like:
```C#
if (string.Equals(CORECLR_SEED, "random", StringComparison.OrdinalIgnoreCase))
{
    seed = new Random().Next();
}
else
{
    seed = DEFAULT_SEED;
}
```

That way `0` (or any other parsable integer value) can still be used, if they are interesting.

Reply:
Ah, right - wasn't thinking about the fact that the environment variable is a string, so you can pass it "random" ;-)

Reply:
I've updated it to do just this for now.